packages/fast-jss-manager-angular/src/design-system.module.ts

```diff
@@ -0,0 +1,17 @@
+import { NgModule } from "@angular/core";
```

Comment:
this typescript file is named with hyphens but in other places we've used camelCases.  What is the standard we're coding too?  I believe for TypeScript it was camelCase?  Correct me if i'm wrong, but, we need to pick a standard and stick with it.

Reply:
The team preference, and now standard is hyphenated not camelCase for ts files.